tests/network-ovn

```diff
@@ -38,6 +38,23 @@ waitInstanceReady() (
   return 1 # Failed.
 )
 
+cloudInitReady() {
```

Comment:
Let's call this setupCloudInitReady, as the former sounds like a statement of fact.

Reply:
We should probably add the profile name as an arg as well as project.